apps/emqx_ft/src/emqx_ft_schema.erl

```diff
@@ -82,13 +84,25 @@ fields(file_transfer) ->
             )},
         {storage,
             mk(
-                hoconsc:union([
-                    ref(local_storage)
-                ]),
+                hoconsc:union(
+                    fun
+                        (all_union_members) ->
+                            [
+                                % NOTE: by default storage is disabled
+                                undefined,
```

Comment:
This feels hacky, yet I didn't found a clearer way to tell hocon "there's no default for this union". This is due to the need to be able to "unset" file transfer storage through config, because for me it felt strange that we're currently unable to "unset" the config after setting it up (see `t_remove_restore_config/1` testcase).

Q: Do we really need that?

Reply:
Having `enable`, do we still need this?

Reply:
I'm leaning to _no, we don't_. This will leave a couple of code paths untested since we'll no longer be able to change storage to something other than `local`, but I guess it's fine price to pay for making configuration schema simpler.